mailpoet/package.json

```diff
@@ -32,7 +32,8 @@
     "**/newsletter_editor/**/*spec.js": "../eslint-config/node_modules/.bin/eslint -c ../eslint-config/.eslintrc.tests_newsletter_editor.json --max-warnings 0",
     "*.php": [
       "phplint",
-      "./do qa:code-sniffer"
+      "./do qa:code-sniffer",
+      "bash -c './do qa:phpstan'"
```

Comment:
Just out of curiosity, what is the reason to use `bash -c './do qa:phpstan'` instead of just `./do qa:phpstan` as we are doing in the line above for PHPCS?

Reply:
@rodrigoprimo I explained it [here](https://github.com/mailpoet/mailpoet/pull/4002/commits/9bb7b86c8a7ae4bcd5e236cebb3c0d01f36aa064), it's because lint-staged always passes the list of files.

Reply:
Ah, thanks. I missed the explanation in the commit message.